internal/commands/push/command_test.go

```diff
@@ -1125,34 +1132,136 @@ func TestPushHandlerCreateNewApp(t *testing.T) {
     }
 }
 `,
-		},
-	} {
-		t.Run(fmt.Sprintf("with a %s config file and nested local path", tc.appConfig.String()), func(t *testing.T) {
-			tmpDir, teardown, tmpDirErr := u.NewTempDir("push_handler")
-			assert.Nil(t, tmpDirErr)
-			defer teardown()
+			},
+		} {
+			t.Run(fmt.Sprintf("and a %s config file", tc.appConfig.String()), func(t *testing.T) {
+				tmpDir, teardown, tmpDirErr := u.NewTempDir("push_handler")
+				assert.Nil(t, tmpDirErr)
+				defer teardown()
 
-			app := local.App{RootDir: tmpDir, Config: tc.appConfig, AppData: tc.appData}
+				app := local.App{RootDir: tmpDir, Config: tc.appConfig, AppData: tc.appData}
 
-			app.WriteConfig()
+				app.WriteConfig()
 
-			out := new(bytes.Buffer)
-			console, ui, consoleErr := mock.NewConsoleWithOptions(mock.UIOptions{AutoConfirm: true}, out)
-			assert.Nil(t, consoleErr)
-			defer console.Close()
+				out := new(bytes.Buffer)
+				console, ui, consoleErr := mock.NewConsoleWithOptions(mock.UIOptions{AutoConfirm: true}, out)
+				assert.Nil(t, consoleErr)
+				defer console.Close()
 
-			cmd := &Command{inputs{LocalPath: filepath.Join(tmpDir, "nested"), RemoteApp: "appID"}}
-			err := cmd.Handler(nil, ui, cli.Clients{Realm: realmClient})
+				cmd := &Command{inputs{LocalPath: filepath.Join(tmpDir, "nested"), RemoteApp: "appID"}}
+				err := cmd.Handler(nil, ui, cli.Clients{Realm: realmClient})
 
-			assert.Nil(t, err)
-			configData, readErr := ioutil.ReadFile(filepath.Join(tmpDir, tc.appConfig.String()))
-			assert.Nil(t, readErr)
-			assert.Equal(t, tc.expectedConfig, string(configData))
+				assert.Nil(t, err)
```

Comment:
[style/nit] I think this was existing code, but should move this assert up one line along with the code that produces the `err` (alternatively, could inline it all: `assert.Nil(t, cmd.Handler(nil, ui, cli.Clients{Realm: realmClient}))`